apps/nextjs-website/src/components/templates/TutorialPageTemplate/TutorialPageTemplate.tsx

```diff
@@ -78,6 +78,21 @@ const TutorialPageTemplate = ({
                 </Box>
               )}
             </Grid>
+            <Grid
+              item
+              xs={false}
+              lg={4}
+              sx={{
+                position: 'sticky',
+                top: 200,
+                maxWidth: '270px !important',
+              }}
+              id='lateral-menu'
+            >
+              <Box>
+                <PartRendererMenu parts={parts ?? []} />
+              </Box>
+            </Grid>
```

Comment:
I see some differences with static generated tutorial `GuideInPageMenu` like positioning and behavior. I cannot make a suggestion out of this beacuse some changes involve also part not touched in this PR but i leave u a snippet of this file with my changes.
```import EContainer from '@/editorialComponents/EContainer/EContainer';
import ProductBreadcrumbs from '@/components/atoms/ProductBreadcrumbs/ProductBreadcrumbs';
import { productPageToBreadcrumbs } from '@/helpers/breadcrumbs.helpers';
import { FragmentProvider } from '@/components/organisms/FragmentProvider/FragmentProvider';
import { Box, Typography } from '@mui/material';
import RelatedLinks from '@/components/atoms/RelatedLinks/RelatedLinks';
import ProductLayout from '@/components/organisms/ProductLayout/ProductLayout';
import { StrapiTutorials } from '@/lib/strapi/tutorial';
import { Product } from '@/lib/types/product';
import PartRenderer from '@/components/molecules/PartRenderer/PartRenderer';
import { Part } from '@/lib/types/part';
import PartRendererMenu from '@/components/molecules/PartRendererMenu/PartRendererMenu';

// TODO: Remove once the migration to CMS contents will be completed
type TutorialPageTemplateProps = {
  readonly bannerLinks?: StrapiTutorials['data'][0]['attributes']['bannerLinks'];
  readonly parts?: readonly Part[];
  readonly path: string;
  readonly product?: Product;
  readonly relatedLinks?: StrapiTutorials['data'][0]['attributes']['relatedLinks'];
  readonly title: string;
};

const TutorialPageTemplate = ({
  parts,
  path,
  product,
  relatedLinks,
  title,
}: TutorialPageTemplateProps) => {
  return (
    <ProductLayout
      product={product}
      path={path}
      // bannerLinks={bannerLinks} // TODO: Enable this property once the bannerLinks refactor will be completed
    >
      {product && (
        <EContainer
          sx={{
            // 80px is the height of the product header
            marginTop: '80px',
            marginX: 'auto',
            paddingTop: 3,
          }}
        >
          <ProductBreadcrumbs
            breadcrumbs={[
              ...productPageToBreadcrumbs(product, path, [
                {
                  name: title,
                  path: path,
                },
              ]),
            ]}
          />
        </EContainer>
      )}
      <FragmentProvider>
        <Box
          sx={{
            display: 'flex',
            flexDirection: { xs: 'column', lg: 'row' },
            maxWidth: '1156px',
            margin: '0 auto',
            paddingY: '56px',
          }}
        >
          <Box
            sx={{
              flexGrow: { lg: 1 },
              maxWidth: {
                xs: '100%',
                lg: '822px',
              },
            }}
          >
            <Typography
              component='h1'
              sx={{
                fontSize: '38px',
                fontWeight: 700,
                lineHeight: '42px',
                paddingY: 2,
              }}
            >
              {title}
            </Typography>
            {parts && (
              <Box mt={1}>
                {parts.map((part, index) => (
                  <PartRenderer key={index} part={part} />
                ))}
              </Box>
            )}
          </Box>
          <Box
            sx={{
              display: { xs: 'none', lg: 'initial' },
              position: 'relative',
              // 78px is the height of the header, 80px is the height of the product header
              paddingTop: '158px',
              paddingLeft: '64px',
              width: { lg: '270px' },
            }}
          >
            <Box
              sx={{
                position: 'sticky',
                maxWidth: '270px',
                top: 144,
              }}
            >
              <PartRendererMenu parts={parts ?? []} />
            </Box>
          </Box>
        </Box>
      </FragmentProvider>
      {relatedLinks && (
        <RelatedLinks
          title={relatedLinks?.title}
          links={relatedLinks?.links ?? []}
        />
      )}
    </ProductLayout>
  );
};

export default TutorialPageTemplate;

```

Reply:
@tommaso1 I think that @MarcoPonchia with "behaviour" is referring to what happens when you click on a link on the side menu: when navigating to an anchor link, the title associated with the anchor is hidden by the main menu. 
We solved this issue on guides and manuals pages with the PR #899, by adding a box around the h* tags with a positive padding top and a negative margin top of the same size and by moving the id to the surrounding box.
Is there a way to do the same when dealing with the titles generated by the Strapi part renderer?